### ISSUE TITLE
[Feature Request] Thumbnail PageBar

### ISSUE THREAD

**Issue by tomchr9** [kanryu/quickviewer#96]
Option to have a thumbnail PageBar at the bottom of the window with all the images in the folder/archive.

Thank you for a great image viewer :-)

**Comment by kanryu** [comment]
When switching the image of the folder, near the slider
Displaying thumbnail images is a function realized with many image viewers.

QuickViewer does not do such behavior, but I understand that demand is high.

**Comment by tomchr9** [comment]
Yes, instead of just displaying the picture number such as PageBar does today, thumbnail preview would make browsing large folders/archives much easier. Something like this:
![Thumbnail preview](https://i.imgur.com/LrW8XRD.jpg)

**Comment by wtfiwinomgs** [comment]
we need this really bad. right now I look at an image I have no idea which is which..
